deployments/common/image/Dockerfile.trailer

```diff
@@ -28,10 +28,10 @@ RUN /opt/common-scripts/fix-certs
 COPY --chown=${NB_UID}:${NB_GID} environments/test    /opt/environments/test
 
 # Install security patches now that build is complete;  potentially this could be done sooner
-RUN apt-get update && \
-    apt-get dist-upgrade --yes && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+# -- RUN apt-get update && \
```

Comment:
I might be useful to add a comment here explaining why these lines are commented out but left in the file.  Ditto for all the mission-specific files.

Reply:
I left them there was as a reminder of what we used to do until our build process stabilizes,  including your scipy-notebook base image build.   I was thinking the base image build would make the above dist-upgrade unnneeded...  but wanted to see it in action first.